cub/cub/util_temporary_storage.cuh

```diff
@@ -73,18 +72,18 @@ _CCCL_HOST_DEVICE _CCCL_FORCEINLINE cudaError_t AliasTemporaries(
   void* d_temp_storage,
   size_t& temp_storage_bytes,
   void* (&allocations)[ALLOCATIONS],
-  size_t (&allocation_sizes)[ALLOCATIONS])
+  const size_t (&allocation_sizes)[ALLOCATIONS])
 {
-  constexpr int ALIGN_BYTES = 256;
-  constexpr int ALIGN_MASK  = ~(ALIGN_BYTES - 1);
+  constexpr size_t ALIGN_BYTES = 256;
+  constexpr size_t ALIGN_MASK  = ~(ALIGN_BYTES - 1);
 
   // Compute exclusive prefix sum over allocation requests
   size_t allocation_offsets[ALLOCATIONS];
   size_t bytes_needed = 0;
   for (int i = 0; i < ALLOCATIONS; ++i)
   {
-    size_t allocation_bytes = (allocation_sizes[i] + ALIGN_BYTES - 1) & ALIGN_MASK;
-    allocation_offsets[i]   = bytes_needed;
+    const size_t allocation_bytes = (allocation_sizes[i] + ALIGN_BYTES - 1) & ALIGN_MASK;
```

Comment:
I changed the types from `int` to `size_t` because that's the type the values promote to in the computations below. Also, the bitflip on a signed int scared me.